merfi/backends/rpm_sign.py

```diff
@@ -39,6 +40,16 @@ def clear_sign(self, path, command):
         """
         logger.info('signing: %s' % path)
         out, err, code = util.run_output(command)
+        # Sometimes rpm-sign will fail with this error. I've opened a ticket
+        # with RH secalert. For now, sleep and retry as a workaround.
+        known_failure = "ERROR: unhandled exception occurred: ('')."
+        tries = 1
```

Comment:
this should be `0`, otherwise the first `tries` count will report as 2 instead of 1.

Reply:
Right, it will be the second try